config/config.go

```diff
@@ -493,8 +493,8 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("datacache.filename", "")
 	v.SetDefault("datacache.cache_size", 0)
 	v.SetDefault("datacache.ttl_seconds", 0)
-	v.SetDefault("category_mapping.filesystem", true)
-	v.SetDefault("category_mapping.directorypath", "./static/category-mapping")
+	v.SetDefault("category_mapping.filesystem.enabled", true)
```

Comment:
We should probably default to false, since this should be set up individually by any PBS host that wants to implement this.